lib/widgets/wallet_list.dart

```diff
@@ -88,6 +88,24 @@ class WalletListState extends State<WalletList> {
       selectedAddressList = walletStorage.currentAddressList;
       walletStorage.wallets.forEach((key, value) {
         walletIdList.add(value.id);
+        walletIdList.add(value.id);
+        walletIdList.add(value.id);
+        walletIdList.add(value.id);
+        walletIdList.add(value.id);
+        walletIdList.add(value.id);
+        walletIdList.add(value.id);
+        walletIdList.add(value.id);
+        walletIdList.add(value.id);
+        walletIdList.add(value.id);
+        walletIdList.add(value.id);
+        walletIdList.add(value.id);
+        walletIdList.add(value.id);
+        walletIdList.add(value.id);
+        walletIdList.add(value.id);
+        walletIdList.add(value.id);
+        walletIdList.add(value.id);
+        walletIdList.add(value.id);
+        walletIdList.add(value.id);
```

Comment:
???

Reply:
It' a trick to debug the wallet list :shushing_face: